storage/innobase/handler/ha_innodb.cc

```diff
@@ -3809,6 +3816,20 @@ static int innodb_init_params()
 		DBUG_RETURN(HA_ERR_INITIALIZATION);
 	}
 
+	MYSQL_SYSVAR_NAME(log_write_ahead_size).max_val = log_sys.buf_size;
```

Comment:
Not an issue for now as innodb_log_buffer_size cannot change dynamically. If we ever try to make it dynamic, the innodb_write_ahead_size and the max_limit needs to be adjusted accordingly.

Reply:
For some future-proofing it would make sense to add a test for `SET GLOBAL innodb_log_buffer_size` with a comment that it must be consistent with `innodb_log_write_ahead_size`.